gslb/rest/dq_nodes.go

```diff
@@ -260,8 +261,28 @@ func (restOp *RestOperations) createOrDeletePathHm(aviGSGraph *nodes.AviGSObject
 				hmKey := avicache.TenantName{Tenant: utils.ADMIN_NS, Name: hmName}
 				restOp.ExecuteRestAndPopulateCache(op, nil, &hmKey, key)
 				if op.Err != nil {
-					gslbutils.Errf("key: %s, hmKey: %v, msg: error while performing rest operation", key, hmKey)
-					return op.Err
+					if !strings.Contains(op.Err.Error(), "Health monitor with this Name and Tenant ref already exists") {
+						gslbutils.Errf("key: %s, hmKey: %v, msg: error while performing rest operation", key, hmKey)
+						return op.Err
+					}
+					// AMKO's UUID has changed, in that case do a GET and PUT operation on health monitor
+					// to update the health monitor with new UUID.
+					hm, err := avicache.GetHMFromName(hmName, false)
```

Comment:
As discussed we need to add a check to see if the previously existing HM was created by the same AMKO as this might an undesired case.

Reply:
in that case, the `hmObj` will not be empty and it wont hit this code path.